app/src/main/kotlin/no/nav/tiltakspenger/vedtak/repository/behandling/ValgteTiltaksdeltagelserDbJson.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.tiltakspenger.vedtak.repository.behandling
+
+import no.nav.tiltakspenger.libs.json.deserialize
+import no.nav.tiltakspenger.libs.json.serialize
+import no.nav.tiltakspenger.libs.periodisering.PeriodeMedVerdi
+import no.nav.tiltakspenger.libs.periodisering.Periodisering
+import no.nav.tiltakspenger.saksbehandling.domene.saksopplysninger.Saksopplysninger
+import no.nav.tiltakspenger.saksbehandling.domene.tiltak.ValgteTiltaksdeltakelser
+import no.nav.tiltakspenger.vedtak.repository.felles.PeriodeDbJson
+import no.nav.tiltakspenger.vedtak.repository.felles.toDbJson
+
+private data class ValgteTiltaksdeltakelserDbJson(
+    val value: List<TiltaksdeltakelsePeriodeMedVerdi>,
+)
+
+private data class TiltaksdeltakelsePeriodeMedVerdi(
+    val periode: PeriodeDbJson,
+    val verdi: String,
```

Comment:
Er det lettere å forstå tiltaksdeltagelseId?

Reply:
Ja, hvis det ikke er noen fast navnekonvensjon for disse periode-med-verdi-objektene kan jeg gjerne endre det :) Må endre i flywayscriptet også da, men det er jo bare kjørt lokalt hos meg uansett :D